sigstore/_cli.py

```diff
@@ -686,9 +686,21 @@ def _sign(args: argparse.Namespace) -> None:
     with signing_ctx.signer(identity) as signer:
         for file, outputs in output_map.items():
             logger.debug(f"signing for {file.name}")
-            with file.open(mode="rb", buffering=0) as io:
+            with file.open(mode="rb", buffering=0) as iof:
                 try:
-                    result = signer.sign(input_=io)
+                    import hashlib
+                    import io
+                    from sigstore import hashes as sigstore_hashes
+                    from sigstore_protobuf_specs.dev.sigstore.common.v1 import HashAlgorithm
```

Comment:
Can we move these imports to the top level? Inline imports like this tend to be a lot harder to test/easy to introduce circularities with 🙂

Reply:
accidental commit that was only used for testing. Removed.